### ISSUE TITLE
Variables defined inside a block crashes lesshint

### ISSUE THREAD

**Issue by trygveaa** [lesshint/lesshint#166]
If you run lesshint on a less file like this:
```
p {
  @mycolor: #000000;
  color: @mycolor;
}
```

It crashes with:
```
An unkown error occured, please file an issue with this info:
TypeError: Cannot read property 'content' of null
    at Object.zeroUnitLinter [as lint] (/home/trygve/dev/hb/frontend/node_modules/lesshint/lib/linters/zero_unit.js:19:61)
    at /home/trygve/dev/hb/frontend/node_modules/lesshint/lib/linter.js:140:32
    at Array.forEach (native)
    at /home/trygve/dev/hb/frontend/node_modules/lesshint/lib/linter.js:126:17
    at Node.traverse (/home/trygve/dev/hb/frontend/node_modules/gonzales-pe/lib/node/basic-node.js:207:5)
    at Node.traverse (/home/trygve/dev/hb/frontend/node_modules/gonzales-pe/lib/node/basic-node.js:212:35)
    at Node.traverse (/home/trygve/dev/hb/frontend/node_modules/gonzales-pe/lib/node/basic-node.js:212:35)
    at Node.traverse (/home/trygve/dev/hb/frontend/node_modules/gonzales-pe/lib/node/basic-node.js:212:35)
    at Object.exports.lint (/home/trygve/dev/hb/frontend/node_modules/lesshint/lib/linter.js:125:9)
    at Lesshint.checkString (/home/trygve/dev/hb/frontend/node_modules/lesshint/lib/lesshint.js:67:25)
```

This is with lesshint 1.5.0. It did not crash with 1.2.2.

**Comment by jwilsson** [comment]
This should've been fixed just this morning with `1.5.1`. Could you check with that version?

**Comment by trygveaa** [comment]
Thanks, it is indeed fixed with `1.5.1`. I didn't notice that release, since it's not tagged yet.

**Comment by jwilsson** [comment]
My bad, I forgot to push the tag :)
